clrdefinitions.cmake

```diff
@@ -195,6 +195,9 @@ add_definitions(-DFEATURE_STRONGNAME_MIGRATION)
 if (CLR_CMAKE_PLATFORM_UNIX OR CLR_CMAKE_TARGET_ARCH_ARM64)
     add_definitions(-DFEATURE_STUBS_AS_IL)
 endif ()
+if (FEATURE_FNV_MEM_OPTIMIZATIONS)
```

Comment:
I think the name of the ifdef should be something like `FEATURE_NGEN_RELOCS_OPTIMIZATIONS` to better describe what it does.

Reply:
And switch the existing places that use `#if defined(PLATFORM_UNIX) && defined(_TARGET_ARM_)` to use it as well.

Reply:
Could we add build with FEATURE_NGEN_RELOCS_OPTIMIZATIONS for Linux ARM to default set of tests on pull requests?

Reply:
That's certainly possible.